tests/workflows/test_rolling_release_workflow.py

```
@@ -0,0 +1,289 @@
+            return [crate for crate in str(lint_value).split() if crate]
+
+
+def _extract_lint_crates_from_text(workflow_text: str) -> list[str]:
+    """Return the list of crate names from workflow text."""
+    yaml = YAML()
+    parsed = yaml.load(workflow_text)
+    parsed_dict = parsed if isinstance(parsed, dict) else {}
+    lint_value = _find_lint_crates_value(parsed_dict)
+
+    if lint_value is None:
+        raise AssertionError(f"{WORKFLOW_PATH} is missing a LINT_CRATES declaration")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)